units/sdlmouse.inc

```diff
@@ -1,260 +1,438 @@
 //from "sdl_mouse.h"
 
 type
-  PSDL_Cursor = Pointer;
+  PSDL_Cursor = Pointer;  {**< Implementation dependent *}
```

Comment:
Please use either:
```
PSDL_Cursor = type Pointer;
```
or
```
PSDL_Cursor = ^TSDL_Cursor;
TSDL_Cursor = record end;
```
The current code makes `PSDL_Cursor` an alias for `Pointer`, which means it's legal to assign **any** pointer to a variable of this type.

Reply:
Will fix this. Thanks.